src/homestars-icons/icons/Star.js

```diff
@@ -1,28 +1,56 @@
 import React from "react";
 import PropTypes from "prop-types";
 import iconWrapper from "../iconWrapper";
+import { brightStar } from "../colours.js";
 
 const propTypes = {
   fill: PropTypes.string,
-  stroke: PropTypes.string
+  showGradient: PropTypes.bool,
+  stroke: PropTypes.string,
+  strokeWidth: PropTypes.number
 };
 
 const defaultProps = {
-  fill: "none"
+  fill: "none",
+  showGradient: false,
+  stroke: "black",
+  strokeWidth: 2
 };
 
-function Star({ fill, stroke }) {
+function Star({ fill, showGradient, stroke, strokeWidth }) {
+  const renderGradient = () => {
+    if (showGradient) {
+      return pug`
+        path(className='mix-blend-overlay', fill='url(#si-a)', fillOpacity='.4', d='M5.15009504,21.8736997 C4.50840631,22.2117997 3.7581967,21.6639527 3.88087248,20.9454904 L4.92047113,14.8503013 C4.96922689,14.5654209 4.8748609,14.2742793 4.66882849,14.0723585 L0.265082318,9.75532427 C-0.253930624,9.24660921 0.0323128766,8.36066235 0.749494396,8.25578878 L6.83610071,7.36671138 C7.12077145,7.3244489 7.36769579,7.14444203 7.49508988,6.88460602 L10.2159759,1.33882916 C10.5368203,0.686108599 11.4631797,0.686108599 11.7840241,1.33882916 L14.5064829,6.88460602 C14.633877,7.14444203 14.8808013,7.3244489 15.165472,7.36671138 L21.2505056,8.25578878 C21.9676871,8.36066235 22.2539306,9.24660921 21.7349177,9.75532427 L17.3311715,14.0723585 C17.1251391,14.2742793 17.0307732,14.5654209 17.0795289,14.8503013 L18.1191275,20.9454904 C18.2418033,21.6639527 17.4931665,22.2117997 16.8514777,21.8736997 L11.4065601,18.9951552 C11.151772,18.8605413 10.848228,18.8605413 10.5934399,18.9951552 L5.15009504,21.8736997 Z')
```

Comment:
`mix-blend-overlay` isn't going to make it to whoever is consuming this component. Better to inline that style.